### PR TITLE
upgrade to openapi v2.0.0

### PR DESCRIPTION
**Pull request by arvindkrishnakumar-okta**
## Description

This PR contains the changes to the Java Management SDK to be able to work with the latest [Open API Spec](https://github.com/okta/openapi) v2.0.0 (Major Release).

This will result in a major release of the Java Management SDK (v2.0.0) 🚀 .

ℹ️ This PR will be merged to dev base branch (_dev_openapi_v2.0.0_major_rel_). All future PRs related to openapi spec change will be merged to this dev base branch. Once ready for release, this dev base branch will be merged to _master_.
  
## Issues \#
[OKTA-295545](https://oktainc.atlassian.net/browse/OKTA-295545)
[OKTA-289669](https://oktainc.atlassian.net/browse/OKTA-289669)

## Covers
- [X] Generate Model objects from new Open API spec
- [X] SDK internal code changes inline with Open API spec change
- [x] Integration Tests (existing ones. New ITs will be added via separate JIRAs)

### CODE REVIEW

**Comment by bdemers** [comment]
My suggestion is to create a branch for the new 2.0 work and merge multiple PRs into that, as 2.0 will be _less_ stable while in development. Once is ready for release merge to master and cut a release :2cents:

**Comment by arvindkrishnakumar-okta** [comment]
> My suggestion is to create a branch for the new 2.0 work and merge multiple PRs into that, as 2.0 will be _less_ stable while in development. Once is ready for release merge to master and cut a release :2cents:

Noted. Created a dev branch - _dev_openapi_v2.0.0_major_rel_ that would hold all openapi changes. This PR would be merged to the dev branch. All future PRs related to openapi will be merged to the dev branch. Once ready for release, the dev branch would be merged to _master_.

File: swagger-templates/src/main/resources/OktaJava/licenseInfo.mustache [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{!
-    Copyright 2017 Okta
+    Copyright 2020-Present Okta, Inc.
```
Comment (bdemers) [inline swagger-templates/src/main/resources/OktaJava/licenseInfo.mustache:2]: Keep the original date. The copyright start date is usually the most important one
```suggestion
    Copyright 2017-Present Okta, Inc.
```

File: swagger-templates/src/main/resources/OktaJavaImpl/licenseInfo.mustache [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {{!
-    Copyright 2017 Okta
+    Copyright 2020-Present Okta, Inc.
```
Comment (bdemers) [inline swagger-templates/src/main/resources/OktaJavaImpl/licenseInfo.mustache:2]:
```suggestion
    Copyright 2017-Present Okta, Inc.
```

**Pull request merged**